ui_tests/src/test/resources/com/saucedemo/brokenTests.feature

```diff
@@ -18,12 +18,12 @@ Feature: Mixed Broken Tests
     Given I open start page
     When I set 'problem_user' as username on Sign in page
     And I set 'secret_sauce' as password on Sign in page
-    And I click 'Sign In' button on Sign in page expecting failure
+    And I click 'Sign In' button on Sign in page
     Then I am on Products page
 
   Scenario Outline: Sign in as <user> user
     Given I open start page
-    When I set '<problem_user>' as username on Sign in page
+    When I set 'problem_user' as username on Sign in page
     And I set 'secret_sauce' as password on Sign in page
```

Comment:
@masrur18  Please make sure that this scenario is doing what it supposed to be doing as per the:
`Scenario Outline: Sign in as <user> user`

Reply:
Done